### DISCUSSION TITLE
Issues with WalletConnect

### DISCUSSION THREAD

**Discussion by arslancodes** [rainbow-me/rainbowkit#1212] [Q&A]
I'm having issues switching chains when connected to a WalletConnect supported wallet (Zerion) on mobile using the Rainbowkit. It works sometimes, and I need to know if it's a known issue with WalletConnect?

To give you context, the following is the message I get at times and something it works flawlessly and switches the network.

![image](https://user-images.githubusercontent.com/35448297/236629449-ad9697a9-0e52-43c1-987d-b45de1923618.png)

**Comment by DanielSinclair** [comment]
If you're still seeing this behavior, please open a new Issue on the repo and we'll investigate.

**Comment by arslancodes** [comment]
The issue has been resolved. There were issues with my code that I fixed. Thanks
